app/services/folio_client.rb

```diff
@@ -135,19 +137,21 @@ def cancel_hold_request(user_id, hold_id)
     response
   end
 
-  # Change hold request date
-  # @example client.change_pickup_location(hold_id: '4a64eccd-3e44-4bb0-a0f7-9b4c487abf61',
+  # TODO: after FOLIO launch rename this method to reflect service point terminology (maybe change_pickup_service_point)
+  #
+  # Change hold request service point
+  # @example client.change_pickup_library(hold_id: '4a64eccd-3e44-4bb0-a0f7-9b4c487abf61',
   #                                        pickup_location_id: 'bd5fd8d9-72f3-4532-b68c-4db88063d16b')
   # @param [String] hold_id the UUID of the FOLIO hold
-  # @param [String] pickup_location_id the uuid of the new location
-  def change_pickup_location(hold_id:, pickup_location_id:)
-    request_data = get_json("/circulation/requests/#{hold_id}")
-    request_data['pickupServicePointId'] = 'bd5fd8d9-72f3-4532-b68c-4db88063d16b'
-    response = put("/circulation/requests/#{hold_id}", json: request_data)
-
+  # @param [String] service_point_id the uuid of the new location
+  def change_pickup_library(id, service_point)
```

Comment:
These parameters no longer match the names of the parameters in the method documentation.